lib/utils/constants.dart

```diff
@@ -1,5 +1,19 @@
+import 'package:flutter/material.dart';
+
 const String pokemonOffSet = '0';
 const String pokemonLimit = '151';
 
 //TODO: added for testing GetPokemonDetailsAction. This will be removed
 const pokemonNameInput = 'jigglypuff';
+
+const pokemonOverviewTitle = Text('Pokemon Overview');
+const gridViewPadding = EdgeInsets.all(10.0);
```

Comment:
seems no need for this 🙂

Reply:
these values were used in pokemon_overview_page.dart

Reply:
yep, I meant no need to declare a constant just immediately place it there 🙂